delta/plugins/storage/src/main/resources/contexts/files.json

```diff
@@ -25,7 +25,8 @@
     "_storage": {
       "@id": "https://bluebrain.github.io/nexus/vocabulary/storage",
       "@type": "@id"
-    }
+    },
+    "_sourceFile": "https://bluebrain.github.io/nexus/vocabulary/fileId"
```

Comment:
It should be `sourceFile` as a suffix.
Can you check the expanded json-ld/n-triples output to see if it is ok ?

Reply:
Done in https://github.com/BlueBrain/nexus/pull/4618/commits/69176fc32eb3a410a5a21ec41c37a66bd03d0699. The expansion works fine, see below

<details>
<summary>JSON LD with sourceFile expanded</summary>
<br>

```json
{
  "@id": "http://delta:8080/v1/resources/d266ifuc81w8t2q/tcqjlehnys34k1b/_/2283170b-6f60-4928-8b51-c5255ce10284",
  "@type": [
    "https://bluebrain.github.io/nexus/vocabulary/File"
  ],
  "https://bluebrain.github.io/nexus/vocabulary/bytes": [
    {
      "@value": 16
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/constrainedBy": [
    {
      "@id": "https://bluebrain.github.io/nexus/schemas/files.json"
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/createdAt": [
    {
      "@type": "http://www.w3.org/2001/XMLSchema#dateTime",
      "@value": "2024-01-08T12:18:54.503877Z"
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/createdBy": [
    {
      "@id": "http://delta:8080/v1/realms/test-kvxjtpxtbuttvjfb/users/digfeugjngcpfclh"
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/deprecated": [
    {
      "@value": false
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/digest": [
    {
      "https://bluebrain.github.io/nexus/vocabulary/algorithm": [
        {
          "@value": "SHA-256"
        }
      ],
      "https://bluebrain.github.io/nexus/vocabulary/value": [
        {
          "@value": "0afef4502ba93acba18d3fa81eb6ae04867ecd3aa886eb5e4df252b86ceaba7f"
        }
      ]
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/filename": [
    {
      "@value": "yschpokcsvrmeuad"
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/incoming": [
    {
      "@id": "http://delta:8080/v1/files/d266ifuc81w8t2q/tcqjlehnys34k1b/http:%2F%2Fdelta:8080%2Fv1%2Fresources%2Fd266ifuc81w8t2q%2Ftcqjlehnys34k1b%2F_%2F2283170b-6f60-4928-8b51-c5255ce10284/incoming"
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/mediaType": [
    {
      "@value": "text/plain; charset=UTF-8"
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/origin": [
    {
      "@id": "Client"
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/outgoing": [
    {
      "@id": "http://delta:8080/v1/files/d266ifuc81w8t2q/tcqjlehnys34k1b/http:%2F%2Fdelta:8080%2Fv1%2Fresources%2Fd266ifuc81w8t2q%2Ftcqjlehnys34k1b%2F_%2F2283170b-6f60-4928-8b51-c5255ce10284/outgoing"
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/project": [
    {
      "@id": "http://delta:8080/v1/projects/d266ifuc81w8t2q/tcqjlehnys34k1b"
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/rev": [
    {
      "@value": 1
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/self": [
    {
      "@id": "http://delta:8080/v1/files/d266ifuc81w8t2q/tcqjlehnys34k1b/http:%2F%2Fdelta:8080%2Fv1%2Fresources%2Fd266ifuc81w8t2q%2Ftcqjlehnys34k1b%2F_%2F2283170b-6f60-4928-8b51-c5255ce10284"
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/sourceFile": [
    {
      "@value": "http://delta:8080/v1/resources/dt3kxjohvk4ldv3/jmnip0odairhgju/_/dhlxiaok32i1w2h"
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/storage": [
    {
      "@id": "https://bluebrain.github.io/nexus/vocabulary/dhq25ddx8cdauda",
      "@type": [
        "https://bluebrain.github.io/nexus/vocabulary/DiskStorage"
      ],
      "https://bluebrain.github.io/nexus/vocabulary/rev": [
        {
          "@value": 1
        }
      ]
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/updatedAt": [
    {
      "@type": "http://www.w3.org/2001/XMLSchema#dateTime",
      "@value": "2024-01-08T12:18:54.503877Z"
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/updatedBy": [
    {
      "@id": "http://delta:8080/v1/realms/test-kvxjtpxtbuttvjfb/users/digfeugjngcpfclh"
    }
  ],
  "https://bluebrain.github.io/nexus/vocabulary/uuid": [
    {
      "@value": "6324ceaf-e153-48c4-b653-4ce408393073"
    }
  ]
}
```

</details>